### PR TITLE
Adds Match CRUD with endpoints

### PR DESCRIPTION
**Pull request by tauil**

### CODE REVIEW

File: migrations/2020-02-25-215149_create_matches/up.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE matches (
```
Comment (frankPairs) [inline migrations/2020-02-25-215149_create_matches/up.sql]: I think it should be idempotent, so you need to check if the table already exists

**Review by frankPairs — APPROVED** [review]
nice!

File: migrations/2020-02-25-215149_create_matches/up.sql (hunk 1 above)
Comment (tauil) [reply migrations/2020-02-25-215149_create_matches/up.sql]: What do you mean? This is according to diesel guides.

Comment (dev-about-code) [reply migrations/2020-02-25-215149_create_matches/up.sql]: `CREATE TABLE  IF NOT EXISTS ...`

Comment (tauil) [reply migrations/2020-02-25-215149_create_matches/up.sql]: Ahhh I see. Thanks. 👍

**Pull request merged**